packages/yourloops/components/alarm/alarms-content-configuration.hook.tsx

```diff
@@ -0,0 +1,234 @@
+      saveInProgress
+  }, [highBg.error, hypoThreshold.error, lowBg.error, nonDataTxThreshold.error, outOfRangeThreshold.error, saveInProgress, veryLowBg.error])
+  const onChange = (
+    value: number,
+    lowValue: number,
+    highValue: number,
+    setValue: React.Dispatch<ValueErrorPair>
+  ): void => {
+    setValue({
+      value,
+      error: isError(value, lowValue, highValue)
+    })
+  }
+
+  const resetToTeamDefaultValues = (): void => {
+    if (!patient) {
+      throw Error('This action cannot be done if the patient is undefined')
+    }
+    const monitoredTeam = PatientUtils.getRemoteMonitoringTeam(patient)
+    const team = teamHook.getTeam(monitoredTeam.teamId)
+    if (!team) {
+      throw Error(`Cannot find team with id ${monitoredTeam.teamId}`)
+    }
+    const defaultMonitoring = team.monitoring
+    if (!defaultMonitoring?.parameters) {
+      throw Error('The given team has no monitoring values')
+    }
+    setHighBg({ ...highBg, value: defaultMonitoring.parameters.highBg })
+    setVeryLowBg({ ...veryLowBg, value: defaultMonitoring.parameters.veryLowBg })
+    setLowBg({ ...lowBg, value: defaultMonitoring.parameters.lowBg })
+    setNonDataTxThreshold({ ...nonDataTxThreshold, value: defaultMonitoring.parameters.nonDataTxThreshold })
+    setOutOfRangeThreshold({ ...outOfRangeThreshold, value: defaultMonitoring.parameters.outOfRangeThreshold })
+    setHypoThreshold({ ...hypoThreshold, value: defaultMonitoring.parameters.hypoThreshold })
+  }
+
+  const save = (): void => {
+    if (
+      lowBg.value !== undefined &&
+      highBg.value !== undefined &&
+      veryLowBg.value !== undefined &&
+      outOfRangeThreshold.value !== undefined &&
+      nonDataTxThreshold.value !== undefined &&
+      hypoThreshold.value !== undefined
+    ) {
+      const reportingPeriod = (monitoring?.parameters?.reportingPeriod && monitoring?.parameters?.reportingPeriod > 0) ? monitoring?.parameters?.reportingPeriod : 55
+      const monitoringUpdated: Monitoring = {
+        enabled: monitoring?.enabled ?? true,
+        status: monitoring?.status,
+        monitoringEnd: monitoring?.monitoringEnd,
+        parameters: {
+          bgUnit: monitoring?.parameters?.bgUnit ?? UNITS_TYPE.MGDL,
+          lowBg: lowBg.value,
+          highBg: highBg.value,
+          outOfRangeThreshold: outOfRangeThreshold.value,
+          veryLowBg: veryLowBg.value,
+          hypoThreshold: hypoThreshold.value,
+          nonDataTxThreshold: nonDataTxThreshold.value,
+          reportingPeriod
+        }
+      }
+      onSave(monitoringUpdated)
+    } else {
+      throw Error('Cannot update team monitoring as some values are not defined')
+    }
+  }
+
+  const onBasicDropdownSelect = (value: string, setValue: React.Dispatch<{ value?: number, error: boolean }>): void => {
+    const valueAsNumber = +value.slice(0, -1)
+    setValue({
+      value: valueAsNumber,
+      error: false
+    })
+  }
+
+  return {
+    isError,
```

Comment:
isError seems exported but never used! It is even more relevant to have it in an util file :slightly_smiling_face: